services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceHelixAdmin.java

```diff
@@ -5332,6 +5348,10 @@ private OfflinePushStatusInfo getOfflinePushStatusInfo(
       // Check whether cluster is in maintenance mode or not
       if (isClusterInMaintenanceMode(clusterName)) {
         moreDetailsBuilder.append("Cluster: ").append(clusterName).append(" is in maintenance mode");
+      } else if (isCanaryRegionPush) {
+        moreDetailsBuilder.append("Version for topic ")
+            .append(kafkaTopic)
+            .append(" isn't created since this is a canary region push");
```

Comment:
can we add the cluster name and fabric name in the log?

Reply:
We can but I prefer to not add the information since `fabric` name is already present in the log and `cluster` seems has not good value?

Sample log
```
INFO [VenicePushJob] [TestNG-method=testCanaryRegionPushJob-1] JobStatusQueryResponse(
    version: 2,
    status: COMPLETED,
    statusDetails: null,
    extraInfo: {dc-0=COMPLETED, dc-1=NOT_CREATED},
    extraDetails: {dc-0=http://adchen-mn1.linkedin.biz:51779 1/1 partitions completed in Da Vinci., dc-1=http://adchen-mn1.linkedin.biz:51880 Offline job hasn't been created yet. and Version for topic testCanaryRegionPushJob_9419d10a6d17b_4a94b9ac_v2 isn't created since this is a canary region push},
```

The `extraDetails` already contains the fabric name.